build.gradle

```diff
@@ -71,9 +71,6 @@ subprojects {
         testLogging.showStandardStreams = true
     }
 
-    sourceCompatibility = 1.8
-    targetCompatibility = 1.8
```

Comment:
We can't have this anymore since the `com.scalar.db.jdk-configuration` plugin sets the JDK for all Java tasks and we need it to be configurable.